- [Got]

(0 chars long)
[stderr]: Traceback (most recent call last):
  File "./main_0.py", line 2, in <module>
    multiply_list_map = __import__('11-multiply_list_map').multiply_list_map
AttributeError: module '11-multiply_list_map' has no attribute 'multiply_list_map'

(237 chars long)
[Expected]
[2, 4, 6, 8, 10]
[1, 2, 3, 4, 5]

(33 chars long)
[stderr]: [Anything]
(0 chars long)
